cmd/metal-api/internal/service/network-service.go

```diff
@@ -439,6 +456,79 @@ func validateAdditionalAnnouncableCIDRs(additionalCidrs []string, privateSuper b
 	return nil
 }
 
+func validatePrefixesAndAddressFamilies(prefixes, destinationPrefixes []string, defaultChildPrefixLength metal.ChildPrefixLength, privateSuper bool) (metal.Prefixes, metal.Prefixes, metal.AddressFamilies, error) {
+
+	pfxs, addressFamilies, err := validatePrefixes(prefixes)
+	if err != nil {
+		return nil, nil, nil, err
+	}
+	// all DestinationPrefixes must be valid and from the same addressfamily
```

Comment:
Is this still correct? Because I see different address families in the destination prefixes in our test environment.

Reply:
removed that comment